xrpl/utils/txn_parser/utils/balance_parser.py

```diff
@@ -0,0 +1,185 @@
+"""Helper functions for balance parser."""
+
+from decimal import Decimal
+from itertools import chain
+from typing import Dict, List, Optional, Union
+
+from xrpl.utils.txn_parser.utils.nodes import NormalizedNode
+from xrpl.utils.txn_parser.utils.types import Balance, BalanceChange, BalanceChanges
+from xrpl.utils.xrp_conversions import drops_to_xrp
+
+
+def _get_xrp_quantity(
+    node: NormalizedNode,
+    value: Optional[Decimal],
+) -> Optional[BalanceChange]:
+    if value is None:
+        return None
+    absolute_value = value.copy_abs()
+    xrp_value = (
+        drops_to_xrp(str(absolute_value)).copy_negate()
+        if value.is_signed()
+        else drops_to_xrp(str(absolute_value))
+    )
+    final_fields = node.get("FinalFields")
+    if final_fields is not None:
+        account = final_fields.get("Account")
+        if account is not None:
+            return BalanceChange(
+                account=account,
+                balance=Balance(  # type: ignore
+                    currency="XRP",
+                    value=f"{xrp_value.normalize():f}",
+                ),
+            )
+    new_fields = node.get("NewFields")
+    if new_fields is not None:
+        account = new_fields.get("Account")
+        if account is not None:
+            return BalanceChange(
+                account=account,
+                balance=Balance(  # type: ignore
```

Comment:
Same question - Why does this need a type ignore?

Reply:
mypy raises: Missing key "issuer" for TypedDict "Balance"